--- pldebugger/Makefile.old	2010-03-12 00:36:55.000000000 +0200
+++ pldebugger/Makefile	2010-03-12 00:38:08.000000000 +0200
@@ -37,11 +37,17 @@
 INSTALL_scripts = pldbgapi.sql
 DOCS		    = README.pldebugger README.plprofiler
 
-subdir       = contrib/debugger
+
+ifdef USE_PGXS
+PGXS = $(shell pg_config --pgxs)
+include $(PGXS)
+else
+subdir = contrib/plantuner
 top_builddir = ../..
 include $(top_builddir)/src/Makefile.global
 include $(top_srcdir)/contrib/contrib-global.mk
-
+endif
+  	 
 ifeq ($(PORTNAME), win32)
 SHLIB_LINK += -lwsock32
 SYMLINKCMD  = cp
