--- line.h.old	2013-12-12 10:47:50.474146641 +0100
+++ line.h	2013-12-12 10:48:13.604975000 +0100
@@ -97,7 +97,7 @@
     \param p2 second returned spherical point
     \return count of found points or <0 if undefined
   */
-  int4 sphereline_latitude_points ( const SLine * sl, float8 lat, SPoint *p1, SPoint *p2 );
+  int sphereline_latitude_points ( const SLine * sl, float8 lat, SPoint *p1, SPoint *p2 );
 
   /*!
     \brief Returns true, if the two lines are equal
--- line.c.old	2013-12-12 10:50:11.699025015 +0100
+++ line.c	2013-12-12 10:50:32.557851591 +0100
@@ -226,11 +226,11 @@
     }
   }
 
-  int4 sphereline_latitude_points ( const SLine * sl, float8 lat, SPoint *p1, SPoint *p2 )
+  int sphereline_latitude_points ( const SLine * sl, float8 lat, SPoint *p1, SPoint *p2 )
   {
 
     float8 inc = sl->theta - floor ( sl->theta / PID ) * PID ;
-    int4   ret = 0;
+    int   ret = 0;
 
     if ( FPgt( lat, PIH ) ) return 0;
     if ( FPlt( lat,-PIH ) ) return 0;
