--- doc/adminguide/slonik_ref.sgml.old	2010-12-14 21:21:04.676861466 +0200
+++ doc/adminguide/slonik_ref.sgml	2010-12-14 21:21:14.569861078 +0200
@@ -2029,7 +2029,7 @@
      node, and an <command>ENABLE_SUBSCRIPTION</command>.
      This means that <xref
      linkend="stmtwaitevent"> must be used following a SUBSCRIBE SET
-     to wait until the last event on the set origin completes.
+     to wait until the last event on the set origin completes.</para>
      </listitem>
 
      <listitem><para> This command has <emphasis>two</emphasis>
