--- pg_filedump-9.0.0/Makefile.old	2011-01-15 12:52:10.850278007 +0200
+++ pg_filedump-9.0.0/Makefile	2011-01-15 12:53:52.200278004 +0200
@@ -1,15 +1,9 @@
 # View README.pg_filedump first
 
 CC=gcc
-CFLAGS=-g -O -Wall -Wmissing-prototypes -Wmissing-declarations
+CFLAGS+=-Wall -Wmissing-prototypes -Wmissing-declarations
 
-INCLUDE=/usr/include/pgsql/server
-
-# PGSQL MUST POINT TO pgsql SOURCE DIRECTORY
-PGSQL=../../../../postgres/pgsql
-
-CRC_SRC=${PGSQL}/src/backend/utils/hash
-CRC_INCLUDE=${PGSQL}/src
+INCLUDE=/usr/pgsql-9.0/include/server/
 
 all: pg_filedump
 
@@ -19,8 +13,8 @@
 pg_filedump.o: pg_filedump.c
 	${CC} ${CFLAGS} -I${INCLUDE} pg_filedump.c -c
 
-pg_crc.o: ${CRC_SRC}/pg_crc.c
-	${CC} ${CFLAGS} -I${CRC_INCLUDE} -I${INCLUDE} ${CRC_SRC}/pg_crc.c -c 
+pg_crc.o: pg_crc.c
+	${CC} ${CFLAGS} -I${INCLUDE} pg_crc.c -c 
 
 clean:
 	rm -rf *.o pg_filedump
--- pg_filedump-9.0.0/Makefile.contrib.old	2011-01-15 12:52:17.454278006 +0200
+++ pg_filedump-9.0.0/Makefile.contrib	2011-01-15 12:53:15.128278007 +0200
@@ -6,7 +6,7 @@
 DOCS = README.pg_filedump
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.0/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
