--- php/geos.c.old	2012-06-08 07:36:41.487999585 -0400
+++ php/geos.c	2012-06-08 07:37:30.991999626 -0400
@@ -48,7 +48,11 @@
 #define zend_function_entry function_entry
 #endif
 
+#if PHP_VERSION_ID < 50399
 static zend_function_entry geos_functions[] = {
+#else
+static zend_function_entry geos_functions[] = {
+#endif
     PHP_FE(GEOSVersion, NULL)
     PHP_FE(GEOSPolygonize, NULL)
     PHP_FE(GEOSLineMerge, NULL)
