--- setup.cfg.old	2013-04-11 13:27:47.775575359 +0300
+++ setup.cfg	2013-04-11 13:27:58.328575007 +0300
@@ -11,7 +11,7 @@
 # build psycopg2. If pg_config is not in the path or is installed under a
 # different name uncomment the following option and set it to the pg_config
 # full path.
-#pg_config=
+pg_config=/usr/pgsql-9.3/bin/pg_config
 
 # Set to 1 to use Python datetime objects for default date/time representation.
 use_pydatetime=1
