diff -Naur pg_filedump-9.1.0.orig/Makefile pg_filedump-9.1.0/Makefile
--- pg_filedump-9.1.0.orig/Makefile	2011-11-26 17:08:46.000000000 -0500
+++ pg_filedump-9.1.0/Makefile	2011-11-26 17:32:51.447513394 -0500
@@ -4,14 +4,11 @@
 FD_VERSION=9.1.0
 
 CC=gcc
-CFLAGS=-g -O -Wall -Wmissing-prototypes -Wmissing-declarations
+CFLAGS+=-Wall -Wmissing-prototypes -Wmissing-declarations
 
-# PGSQL MUST POINT TO pgsql SOURCE DIRECTORY
-PGSQL=../../pgsql
+CRC_SRC_DIR=.
 
-CRC_SRC_DIR=${PGSQL}/src/backend/utils/hash
-
-INCLUDE_DIR=${PGSQL}/src/include
+INCLUDE_DIR=/usr/include/pgsql/server
 
 DISTFILES= README.pg_filedump Makefile Makefile.contrib \
 	pg_filedump.h pg_filedump.c
