--- Makefile.old	2010-11-11 15:08:26.670579208 +0200
+++ Makefile	2010-11-11 15:08:55.360575585 +0200
@@ -9,11 +9,18 @@
 VERSION = $(shell ./pgloader.py --version |cut -d' ' -f3)
 SHORTVER= $(shell ./pgloader.py --version |cut -d' ' -f3 |cut -d '~' -f1)
 
+# for RPMs
+ifdef RPMDESTDIR
+
+libdir = $(LIBDIR)
+exdir =  $(EXDIR)
+else
 # debian setting
 DESTDIR =
 
 libdir   = $(DESTDIR)/usr/share/python-support/pgloader
 exdir    = $(DESTDIR)/usr/share/doc/pgloader
+endif
 
 pgloader = pgloader.py
 examples = examples
