--- Makefile.old	2012-04-03 11:36:22.433877157 +0300
+++ Makefile	2012-04-03 11:38:01.496879417 +0300
@@ -19,7 +19,8 @@
 	$(CC) $(CFLAGS) $(repmgr_OBJS) $(PG_LIBS) $(LDFLAGS) $(LDFLAGS_EX) $(LIBS) -o repmgr
 
 ifdef USE_PGXS
-PGXS := $(shell pg_config --pgxs)
+PG_CONFIG	= /usr/pgsql-9.4devel/bin/pg_config
+PGXS   := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
 subdir = contrib/repmgr
@@ -31,8 +32,8 @@
 # XXX: Try to use PROGRAM construct (see pgxs.mk) someday. Right now
 # is overriding pgxs install.
 install:
-	$(INSTALL_PROGRAM) repmgrd$(X) '$(DESTDIR)$(bindir)'
-	$(INSTALL_PROGRAM) repmgr$(X) '$(DESTDIR)$(bindir)'
+	$(INSTALL_PROGRAM) repmgrd$(X) '$(DESTDIR)$(bindir)/'
+	$(INSTALL_PROGRAM) repmgr$(X) '$(DESTDIR)$(bindir)/'
 
 ifneq (,$(DATA)$(DATA_built))
 	@for file in $(addprefix $(srcdir)/, $(DATA)) $(DATA_built); do \
