--- plr--8.3.0.13.sql.old	2012-10-09 07:45:12.228969707 +0300
+++ plr--8.3.0.13.sql	2012-10-09 07:46:26.141967243 +0300
@@ -9,39 +9,39 @@
 CREATE OR REPLACE FUNCTION plr_version ()
 RETURNS text
 AS 'MODULE_PATHNAME','plr_version'
-LANGUAGE 'C';
+LANGUAGE 'c';
 
 CREATE OR REPLACE FUNCTION reload_plr_modules ()
 RETURNS text
 AS 'MODULE_PATHNAME','reload_plr_modules'
-LANGUAGE 'C';
+LANGUAGE 'c';
 
 CREATE OR REPLACE FUNCTION install_rcmd (text)
 RETURNS text
 AS 'MODULE_PATHNAME','install_rcmd'
-LANGUAGE 'C' WITH (isstrict);
+LANGUAGE 'c' WITH (isstrict);
 REVOKE EXECUTE ON FUNCTION install_rcmd (text) FROM PUBLIC;
 
 CREATE OR REPLACE FUNCTION plr_singleton_array (float8)
 RETURNS float8[]
 AS 'MODULE_PATHNAME','plr_array'
-LANGUAGE 'C' WITH (isstrict);
+LANGUAGE 'c' WITH (isstrict);
 
 CREATE OR REPLACE FUNCTION plr_array_push (_float8, float8)
 RETURNS float8[]
 AS 'MODULE_PATHNAME','plr_array_push'
-LANGUAGE 'C' WITH (isstrict);
+LANGUAGE 'c' WITH (isstrict);
 
 CREATE OR REPLACE FUNCTION plr_array_accum (_float8, float8)
 RETURNS float8[]
 AS 'MODULE_PATHNAME','plr_array_accum'
-LANGUAGE 'C';
+LANGUAGE 'c';
 
 CREATE TYPE plr_environ_type AS (name text, value text);
 CREATE OR REPLACE FUNCTION plr_environ ()
 RETURNS SETOF plr_environ_type
 AS 'MODULE_PATHNAME','plr_environ'
-LANGUAGE 'C';
+LANGUAGE 'c';
 
 REVOKE EXECUTE ON FUNCTION plr_environ() FROM PUBLIC;
 
@@ -78,23 +78,23 @@
 CREATE OR REPLACE FUNCTION plr_set_rhome (text)
 RETURNS text
 AS 'MODULE_PATHNAME','plr_set_rhome'
-LANGUAGE 'C' WITH (isstrict);
+LANGUAGE 'c' WITH (isstrict);
 REVOKE EXECUTE ON FUNCTION plr_set_rhome (text) FROM PUBLIC;
 
 CREATE OR REPLACE FUNCTION plr_unset_rhome ()
 RETURNS text
 AS 'MODULE_PATHNAME','plr_unset_rhome'
-LANGUAGE 'C';
+LANGUAGE 'c';
 REVOKE EXECUTE ON FUNCTION plr_unset_rhome () FROM PUBLIC;
 
 CREATE OR REPLACE FUNCTION plr_set_display (text)
 RETURNS text
 AS 'MODULE_PATHNAME','plr_set_display'
-LANGUAGE 'C' WITH (isstrict);
+LANGUAGE 'c' WITH (isstrict);
 REVOKE EXECUTE ON FUNCTION plr_set_display (text) FROM PUBLIC;
 
 CREATE OR REPLACE FUNCTION plr_get_raw (bytea)
 RETURNS bytea
 AS 'MODULE_PATHNAME','plr_get_raw'
-LANGUAGE 'C' WITH (isstrict);
+LANGUAGE 'c' WITH (isstrict);
 
