--- pkg/pgadmin3.desktop.old	2010-09-20 10:31:35.907286936 +0300
+++ pkg/pgadmin3.desktop	2010-09-20 10:36:49.491215628 +0300
@@ -1,10 +1,10 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=pgAdmin III
-Exec=/usr/bin/pgadmin3
-Icon=/usr/share/pgadmin3/pgAdmin3.png
+Exec=/usr/bin/pgadmin3_91
+Icon=/usr/share/pgadmin3_91/pgadmin3_91.xpm
 Type=Application
 Categories=Application;Development;
 MimeType=text/html
-DocPath=/usr/share/pgadmin3/docs/en_US/index.html
+DocPath=/usr/pgsql-9.1/share/pgadmin3/docs/en_US/index.html
 Comment=PostgreSQL Tools
