--- src/interfaces/libpq/fe-connect.c.old	2010-04-20 16:12:32.398337391 +0300
+++ src/interfaces/libpq/fe-connect.c	2010-04-20 16:13:28.956348563 +0300
@@ -25,7 +25,7 @@
 #include "libpq-fe.h"
 #include "libpq-int.h"
 #include "fe-auth.h"
-#include "pg_config_paths.h"
+#include "../../port/pg_config_paths.h"
 
 #ifdef WIN32
 #include "win32.h"
--- src/interfaces/libpq/fe-misc.c.old	2010-04-20 16:12:36.522337315 +0300
+++ src/interfaces/libpq/fe-misc.c	2010-04-20 16:13:35.266336011 +0300
@@ -57,7 +57,7 @@
 #include "libpq-int.h"
 #include "pqsignal.h"
 #include "mb/pg_wchar.h"
-#include "pg_config_paths.h"
+#include "../../port/pg_config_paths.h"
 
 
 static int	pqPutMsgBytes(const void *buf, size_t len, PGconn *conn);
