--- Makefile.old	2012-03-23 14:54:18.393420878 +0200
+++ Makefile	2012-03-23 14:54:34.567421248 +0200
@@ -13,7 +13,7 @@
 endif
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.0/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
--- lib/Makefile.old	2012-03-23 14:55:13.516422136 +0200
+++ lib/Makefile	2012-03-23 14:55:20.379422292 +0200
@@ -19,7 +19,7 @@
 endif
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.0/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
--- bin/Makefile.old	2012-03-23 14:54:50.500421611 +0200
+++ bin/Makefile	2012-03-23 14:55:02.299421880 +0200
@@ -25,7 +25,7 @@
 endif
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.0/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
