--- cmake/FindPostgreSQL.cmake.old	2013-09-02 18:45:12.122903519 +0300
+++ cmake/FindPostgreSQL.cmake	2013-09-02 18:46:05.328709679 +0300
@@ -21,12 +21,14 @@
 
     find_program(POSTGRESQL_EXECUTABLE NAMES postgres
         PATHS
+	/usr/pgsql-9.3/bin/
         /usr/lib/postgresql/*/bin/
         )
     message(STATUS "POSTGRESQL_EXECUTABLE is " ${POSTGRESQL_EXECUTABLE})
 
     find_program(POSTGRESQL_PG_CONFIG NAMES pg_config
         PATHS
+	/usr/pgsql-9.3/bin/
         /usr/lib/postgresql/*/bin/
         )
     message(STATUS "POSTGRESQL_PG_CONFIG is " ${POSTGRESQL_PG_CONFIG})
@@ -47,6 +49,7 @@
 
     find_path(POSTGRESQL_INCLUDE_DIR postgres.h
         ${T_POSTGRESQL_INCLUDE_DIR}
+	/usr/pgsql-9.3/include/server
         /usr/include/server
         /usr/include/pgsql/server
         /usr/local/include/pgsql/server
