--- configure.old	2010-03-10 01:17:16.000000000 -0800
+++ configure	2010-03-10 01:17:57.000000000 -0800
@@ -6271,13 +6271,13 @@
 	XML2_MINOR=`expr ${XML2_VERSION} : '[0-9]*\.\([0-9]*\)'`
 	XML2_REVISION=`expr ${XML2_VERSION} : '[0-9]*\.[0-9]*\.\([0-9]*\)'`
 		if test "$XML2_MAJOR" -lt 2; then
-		as_fn_error "libxml2 2.6.18 or newer is required to build pgAdmin" "$LINENO" 5
+		as_fn_error "libxml2 2.6.16 or newer is required to build pgAdmin" "$LINENO" 5
 	fi
 	if test "$XML2_MAJOR" -eq 2 && test "$XML2_MINOR" -lt 6; then
-		as_fn_error "libxml2 2.6.18 or newer is required to build pgAdmin" "$LINENO" 5
+		as_fn_error "libxml2 2.6.16 or newer is required to build pgAdmin" "$LINENO" 5
 	fi
-	if test "$XML2_MAJOR" -eq 2 && test "$XML2_MINOR" -eq 6 && test "$XML2_REVISION" -lt 18; then
-		as_fn_error "libxml2 2.6.18 or newer is required to build pgAdmin" "$LINENO" 5
+	if test "$XML2_MAJOR" -eq 2 && test "$XML2_MINOR" -eq 6 && test "$XML2_REVISION" -lt 16; then
+		as_fn_error "libxml2 2.6.16 or newer is required to build pgAdmin" "$LINENO" 5
 	fi
 	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: ok" >&5
 $as_echo "ok" >&6; }
