--- gridsql-2.0/gridsql_env.sh.old	2010-03-08 21:49:40.000000000 +0200
+++ gridsql-2.0/gridsql_env.sh	2010-03-08 21:50:28.000000000 +0200
@@ -15,12 +15,8 @@
 ##########################################################################
 
 # ROOT directory of where GridSQL is installed
-export GSPATH=/usr/local/gridsql-2.0
+export GSPATH=/usr
 
 # Add any referenced JDBC drivers here, like EnterpriseDB's driver
-export CLASSPATH=$GSPATH/lib/edb-jdbc14.jar:$CLASSPATH
-
-# Include scripts in user path
-export PATH=$GSPATH/bin:$PATH
-
+export CLASSPATH=/usr/share/java/edb-jdbc14.jar:$CLASSPATH
 
