--- gridsql-agent-2.0/bin/gs-agent.sh.old	2010-03-10 15:25:49.000000000 +0200
+++ gridsql-agent-2.0/bin/gs-agent.sh	2010-03-10 15:26:42.000000000 +0200
@@ -13,19 +13,12 @@
 
 EXECCLASS=com.edb.gridsql.util.XdbAgent
 
-DIRNAME=`dirname $0`
-
-if [ -f $DIRNAME/../gridsql_env.sh -a -z "$GSPATH" ]
-then
-      source $DIRNAME/../gridsql_env.sh
-fi
-
-GSCONFIG=$GSPATH/config/gridsql_agent.config
-
-# Adjust these if more memory is required
-MINMEMORY=256M
-MAXMEMORY=256M
-
-
-nohup java -classpath ${GSPATH}/bin/xdbengine.jar:${GSPATH}/bin/xdbserver.jar:${GSPATH}/lib/edb-jdbc14.jar:${GSPATH}/lib/log4j.jar:${CLASSPATH} -Xms${MINMEMORY} -Xmx${MAXMEMORY} -Dconfig.file.path=${GSCONFIG} $EXECCLASS $* > ${GSPATH}/log/agent.log 2>&1 &
-
+source /usr/bin/gridsql_env.sh
+ 
+GSCONFIG=/etc/gridsql/gridsql_agent.config
+ 
+ # Adjust these if more memory is required
+ MINMEMORY=256M
+ MAXMEMORY=256M
+ 
+nohup java -classpath /usr/share/java/postgresql-jdbc.jar:/usr/share/java/xdbengine.jar:/usr/share/java/xdbutil.jar:/usr/share/java/edb-jdbc14.jar:/usr/share/java/log4j.jar:${CLASSPATH} -Xms${MINMEMORY} -Xmx${MAXMEMORY} -Dconfig.file.path=${GSCONFIG} $EXECCLASS $* > /var/log/gridsql/agent.log 2>&1 &
