--- Makefile.old	2013-12-12 11:25:15.848553908 +0100
+++ Makefile	2013-12-12 11:25:22.412502669 +0100
@@ -36,9 +36,7 @@
 crushtest: installcheck
 
 pg_sphere.sql.in : $(addsuffix .in, $(PGS_SQL))
-	echo 'BEGIN;' > $@
 	for i in $+ ; do $(AWK) -v pg_version=$(PGVERSION) -f sql.awk < $$i >> $@ ; done
-	echo 'COMMIT;' >> $@
 
 sscan.o : sparse.c
 
@@ -59,4 +57,4 @@
 dist : clean sparse.c sscan.c
 	find . -name '*~' -type f -exec rm {} \;
 	cd .. && tar  --exclude CVS -czf pg_sphere.tar.gz pg_sphere && cd -
- 	
\ No newline at end of file
+ 	
