--- Makefile.old	2012-11-14 14:47:44.924414751 +0000
+++ Makefile	2012-11-14 14:48:38.306455128 +0000
@@ -4,12 +4,12 @@
 FD_VERSION=9.2.0
 
 CC=gcc
-CFLAGS=-g -O -Wall -Wmissing-prototypes -Wmissing-declarations
+CFLAGS+=-Wall -Wmissing-prototypes -Wmissing-declarations
 
 # If working with a PG source directory, point PGSQL_INCLUDE_DIR to its
 # src/include subdirectory.  If working with an installed tree, point to
 # the server include subdirectory, eg /usr/local/include/postgresql/server
-PGSQL_INCLUDE_DIR=../../pgsql/src/include
+PGSQL_INCLUDE_DIR=/usr/pgsql-9.2/include/server
 
 
 DISTFILES= README.pg_filedump Makefile Makefile.contrib \
--- Makefile.contrib.old	2012-11-14 14:48:59.547470371 +0000
+++ Makefile.contrib	2012-11-14 14:49:14.263482267 +0000
@@ -4,7 +4,7 @@
 DOCS = README.pg_filedump
 
 ifdef USE_PGXS
-PG_CONFIG = pg_config
+PG_CONFIG = /usr/pgsql-9.2/bin/pg_config
 PGXS := $(shell $(PG_CONFIG) --pgxs)
 include $(PGXS)
 else
