--- CMakeLists.txt.old	2012-11-09 13:41:55.130073613 +0200
+++ CMakeLists.txt	2012-11-09 13:42:04.536082802 +0200
@@ -26,7 +26,7 @@
   MESSAGE(FATAL ERROR " Please check your Boost installation ") 
 ENDIF(Boost_INCLUDE_DIRS)
 
-EXEC_PROGRAM(pg_config
+EXEC_PROGRAM(/usr/pgsql-9.1/bin/pg_config
  ARGS --pkglibdir
  OUTPUT_VARIABLE LIB_DIR)
 
--- cmake/FindPostgreSQL.cmake.old	2012-11-09 14:01:26.547078521 +0200
+++ cmake/FindPostgreSQL.cmake	2012-11-09 14:01:55.099121211 +0200
@@ -20,6 +20,7 @@
 #  find_path(POSTGRESQL_INCLUDE_DIR libpq-fe.h
 
  FIND_PATH(POSTGRESQL_INCLUDE_DIR postgres.h
+      /usr/pgsql-9.1/include/server
       /usr/include/server
       /usr/include/pgsql/server
       /usr/local/include/pgsql/server
@@ -33,6 +34,7 @@
 
   find_library(POSTGRESQL_LIBRARIES NAMES pq libpq
      PATHS
+     /usr/pgsql-9.1/lib
      /usr/lib
      /usr/local/lib
      /usr/lib/postgresql
